### DISCUSSION TITLE
How to export the RNA expression data of the clustered cells

### DISCUSSION THREAD

**Discussion by Madeinheavena** [satijalab/seurat#4573] [Q&A] [answered]
I have performed this command in R and this is the step I did(below)

>SCGM_0d <- FindNeighbors(SCGM_0d, dims = 1:30)
>SCGM_0d <- RunUMAP(SCGM_0d, dims = 1:30)
>SCGM_0d <- RunTSNE(SCGM_0d, dims = 1:30)
>SCGM_0d <- FindClusters(SCGM_0d, resolution = 0.25)
>saveRDS(SCGM_0d, file = "SCGM_0d-clean-data-Normalizetion-PCA-UMAP-SNE.rds")

I get this rds and I want to get the RNA expression data of two clustered cells from the rds file, 
![28BM@Y(N` 7V4 QW03W@726](https://user-images.githubusercontent.com/82137838/120662399-b2a4b800-c4bb-11eb-8362-8e0f62688470.png)
but I dont konw how to get the data and export it.
help please!

**Comment by timoast** [comment, marked as answer]
You can extract data from the Seurat object using the `GetAssayData()` function. See documentation here: https://rdrr.io/cran/SeuratObject/man/AssayData.html

You can also find the average expression of each gene for groups of cells using the `AverageExpression()` function

**Reply by Madeinheavena** [reply]
Thank you very much!
